frontend/packages/dev-console/src/components/formik-fields/multi-column-field/MultiColumnFieldRow.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { MinusCircleIcon } from '@patternfly/react-icons';
+
+export interface MultiColumnFieldRowProps {
+  name: string;
+  rowIndex: number;
+  children: React.ReactNode;
+  onDelete: () => void;
+}
+
+const MultiColumnFieldRow: React.FC<MultiColumnFieldRowProps> = ({
+  name,
+  rowIndex,
+  onDelete,
+  children,
+}) => (
+  <div className="odc-multi-column-field__row">
+    {React.Children.map(children, (child: React.ReactElement) => {
```

Comment:
Do we need to supply our own onChange handlers to the inputs?

Reply:
No, you will be using formik fields which handles everything for you.